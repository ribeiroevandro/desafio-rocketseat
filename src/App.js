import React, { Component } from 'react';

class App extends Component {
  render() {
    return (
      <div className="App">
        <h1>Quero uma Caneca da Rockeat Seat</h1>
      </div>
    );
  }
}

export default App;
