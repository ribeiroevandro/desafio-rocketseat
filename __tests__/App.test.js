import React from 'react';
import { shallow } from 'enzyme';

import App from '../src/App';

describe('Testing App Component', () => {
  it('Should render correctly', () => {
    const wrapper = shallow(
      <App />
    );

    expect(wrapper).toMatchSnapshot();
  });
});