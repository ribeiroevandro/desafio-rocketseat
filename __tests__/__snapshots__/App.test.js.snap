// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App Component Should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": <h1>
        Quero uma Caneca da Rockeat Seat
</h1>,
      "className": "App",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": "Quero uma Caneca da Rockeat Seat",
      },
      "ref": null,
      "rendered": "Quero uma Caneca da Rockeat Seat",
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": <h1>
          Quero uma Caneca da Rockeat Seat
</h1>,
        "className": "App",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "Quero uma Caneca da Rockeat Seat",
        },
        "ref": null,
        "rendered": "Quero uma Caneca da Rockeat Seat",
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
